web/client/reducers/mapInfo.js

```diff
@@ -344,7 +345,10 @@ function mapInfo(state = initState, action) {
     case MAP_CONFIG_LOADED: {
         return {
             ...state,
-            configuration: action.config.mapInfoConfiguration || state.configuration || {}
+            configuration: {
+                ...state.configuration,
+                ...action.config.mapInfoConfiguration
+            }
```

Comment:
i think here you are changing a little the behaviour.
can you explain me which use case the new code covers?

MAP_CONFIG_LOADED is used when the configuration of the map is loaded and I think it should replace the whole current state of the mapinfo.configuration @offtherailz do you confirm?

Reply:
When we have properties in mapInfo, and then action.config.mapInfoConfguration comes in, in this case all previous properties will be deleted. So at the moment, when action.config.mapInfoConfguration comes, it merges with the previous properties

Reply:
It makes sense.